reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -425,9 +426,10 @@ class StaticHtmlReporter : Reporter() {
     private fun TBODY.evaluatorRow(error: ResolvableIssue) {
         val cssClass = when {
             error.isResolved -> "ort-resolved"
-            else -> "ort-error"
+            error.severity == Severity.ERROR -> "ort-error"
+            else -> "ort-warning"
```

Comment:
I'd prefer something more verbose here that captures all enum values. Like:

        val cssClass = if (error.isResolved) {
            "ort-resolved"
        } else {
            when (error.severity) {
                Severity.ERROR -> "ort-error"
                Severity.WARNING -> "ort-warning"
                Severity.HINT -> throw NotImplementedError("Displaying hints is not implemented yet.")
            }
        }

Reply:
done